### DISCUSSION TITLE
Does it support React?

### DISCUSSION THREAD

**Discussion by fullofcaffeine** [emacs-eaf/emacs-application-framework#857] [Q&A] [answered]
Just found this project, pretty amazed by it! Looks great! Excited to start using it to augment my setup :)

The README seems to suggest it only supports Vue.js. Is that the case? Would it support React apps out-of-the-box or would it require changes to the architecture?

Thanks!

**Comment by MatthewZMD** [comment]
Do you wish to extend EAF using React? This is what EAF means by "EAF supporting Vue.js", it is currently not supported. I believe changes to the architecture isn't necessary, just needing some tweaks similar to Vue.js to make React work. I don't know React well enough to say for certain.

If you mean opening React websites or developing ordinary React apps and preview it using EAF, yes that's already possible, EAF Browser is just like any ordinary modern browser.

**Reply by fullofcaffeine** [reply]
I meant extending EAF using React, indeed. Sorry if I wasn't clear.

**Comment by manateelazycat** [comment, marked as answer]
yes，you can use npm build react template, then make EAF can found index.html that generate by npm build.

**Reply by fullofcaffeine** [reply]
Nice! Thanks!
